addons/web/models/ir_http.py

```diff
@@ -23,7 +23,7 @@
 Multiple debug modes can be activated simultaneously, separated with a
 comma (eg: 'tests, assets').
 """
-ALLOWED_DEBUG_MODES = ['', '1', 'assets', 'tests', 'disable-t-cache']
+ALLOWED_DEBUG_MODES = ['', '1', 'assets', 'tests', 'disable-t-cache', 'use-papermuncher']
```

Comment:
simpler? but we gotta be consistent about that dash between `paper` and `muncher`. Either we use the dash everywhere, either nowhere. But let's not *something* use a dash, and sometime not.
```suggestion
ALLOWED_DEBUG_MODES = ['', '1', 'assets', 'tests', 'disable-t-cache', 'papermuncher']
```